apps/web/src/services/dashboards/widgets/_components/WidgetViewModeSidebar.vue

```diff
@@ -107,7 +104,7 @@ const state = reactive({
 /* Util */
 const updateDashboardWidgetStore = () => {
     // update widget info in dashboard detail store
-    const widgetInfo: DashboardLayoutWidgetInfo = cloneDeep(widgetFormState.widgetInfo);
+    const widgetInfo = cloneDeep(widgetFormState.widgetInfo) as DashboardLayoutWidgetInfo;
```

Comment:
Question: Why did you change type location??

Reply:
There was a type error because the type of `widgetFormState.widgetInfo` is an **optional** `DashboardLayoutWidgetInfo`